modules/acquia_cms_headless/modules/acquia_cms_headless_ui/acquia_cms_headless_ui.routing.yml

```diff
@@ -0,0 +1,24 @@
+acquia_cms_headless_ui.frontpage:
```

Comment:
@modularlab Any reason we need to create this route ? We can use event_subscriber for this (i.e KernelEvents::REQUEST) ?